docs/articles/documentation/recipes/using-page-model.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: docs
+title: Using Page Model
+permalink: /documentation/recipes/using-page-model.html
+---
+# Using Page Model
+
+Page Model is a test automation pattern that allows you to create an abstraction of the tested page
+and use it in test code to refer to page elements.
+
+## Why Use Page Model
+
+Consider the following fixture with a test that types and edits the developer name on the [example](https://devexpress.github.io/testcafe/example/) webpage.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `My fixture`
+    .page `https://devexpress.github.io/testcafe/example/`;
+
+const nameInput = Selector('#developer-name');
+
+test('Text typing basics', async t => {
```

Comment:
Anyway, repeating a constant three or four times in a row is too weird even for demonstrational purposes IMO. Even someone who has no idea about TestCafe selectors will immediately introduce a string constant

```js
const nameInput = '#developer-name';
```

Otherwise, as a reader, I would feel like I'm disrespected a bit.